### DISCUSSION TITLE
Gdu cant open a 7.5GB report file

### DISCUSSION THREAD

**Discussion by anonymousti** [dundee/gdu#151] [General]
I'm trying to open 7.5GB json report file, but the app is unable to display anything. I see a Killed message ate the footer. 
Any idea here ?

**Comment by danie-dejager** [comment]
How did you create the report file? Do you have some way to share some of the unique attributes from the drives? Lots of small files in many directories? Does the report span multiple partitions? I'm assuming you are using the latest  release? What OS?

**Comment by dundee** [comment]
Seems like Out of Memory problem. How much total memory and approx free memory your machine has?

Problems like this would be mitigated by the usage of some persistent key-value storage instead of loading the whole data into memory - https://github.com/dundee/gdu/issues/118. But it will still take some time to implement this.
